packages/actor-init-sparql/lib/HttpServiceSparqlEndpoint.ts

```diff
@@ -156,12 +191,12 @@ Options:
    * @param {module:http.IncomingMessage} request Request object.
    * @param {module:http.ServerResponse} response Response object.
    */
+
   public async handleRequest(engine: ActorInitSparql, variants: { type: string; quality: number }[],
     stdout: Writable, stderr: Writable,
     request: http.IncomingMessage, response: http.ServerResponse): Promise<void> {
-    const mediaType: string = request.headers.accept && request.headers.accept !== '*/*' ?
-      // eslint-disable-next-line multiline-ternary
-      require('negotiate').choose(variants, request)[0].type : null;
+    // Negotiate the best mediatype format
```

Comment:
Comment can be removed, since the method name already describes well what will happen.